community/contributing/content_guidelines.rst

```diff
@@ -0,0 +1,95 @@
+.. _doc_content_guidelines:
+
+Content guidelines
+==================
+
+This document is here to help us assess what we should include in the official
+documentation. Below, you will find a couple of principles and recommendations
+to write accessible content.
+
+We want to achieve two goals:
+
+1. **Empathize with our users**. We should write in a way that makes it easy for
```

Comment:
Leaving this here for debate: Should the punctuation be included in the bold markup when the full sentence is bold? I.e.
```suggestion
1. **Empathize with our users.** We should write in a way that makes it easy for
```

I don't know how much of a reference this is but https://www.otago.ac.nz/writing-style/grammar-punctuation/otago635506.html states:

> Use bold sparingly to emphasise text you wish to stand out such as dates, names, deadlines, etc. Punctuation which follows the bold text should not be bold (unless the whole sentence is in bold).

That's what I tend to do myself, so I don't feel very strongly one way or the other.

Reply:
@NathanLovato I made a suggestion here as an example, but it applies to the whole document (and #4067).
There are more occurrences of complete bold sentences where the punctuation should be bold.